policyengine_us/variables/gov/states/or/tax/income/liheap/or_liheap_benefit_level.py

```diff
@@ -0,0 +1,34 @@
+from policyengine_us.model_api import *
+
+
+class or_liheap_benefit_level(Variable):
+    value_type = int
+    entity = SPMUnit
+    label = "Income range for Oregon LIHEAP eligibility"
+    definition_period = YEAR
+    reference = "https://www.oregon.gov/ohcs/energy-weatherization/Documents/2021-Energy-Assistance-Manual.pdf#page=55"
+
+    defined_for = StateCode.OR
+
+    def formula(spm_unit, period, parameters):
+        income = add(spm_unit, period, ["irs_gross_income"])
+        threshold = spm_unit("or_liheap_income_threshold", period)
+        p = parameters(period).gov.states["or"].liheap
```

Comment:
```suggestion
        p = parameters(period).gov.states.or.liheap
```
Same for other parameters used in formula.